src/assignement2/StorgeClass.java

```diff
@@ -0,0 +1,50 @@
+package assignement2;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.ObjectOutputStream;
+import java.io.ObjectInputStream;
+import java.io.IOException;
+import java.util.Set;
+import java.util.TreeSet;
+/*
+ * This class is use to store all the users data
+ * in the file sytsem
+ */
+
+public class StorgeClass {
+    /*
+     * it takes data from set amd store in file
+     * before exiting
+     * @param us set of user
+     */
+    public void saveChanges(final Set<User> us) {
+        try {
+            FileOutputStream file = new FileOutputStream("userdata.txt");
```

Comment:
file name should be in static constant of  StorgeClass class

Reply:
made it static constant